moped-editor/src/views/projects/projectView/ProjectPhase/CurrentPhaseDeleteModal.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+} from "@mui/material";
+
+const CurrentPhaseDeleteModal = ({ isOpen, setIsOpen }) => {
+  const handleClose = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={handleClose}
+      aria-labelledby="alert-dialog-title"
```

Comment:
this is a tiny detail that caught my eye because of recent accessibility discussion but I don't think there is matching id with this name. I would be okay with removing.